plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/Drilldown.js

```diff
@@ -40,9 +40,16 @@ Ext.define('NX.controller.Drilldown', {
 
   permission: undefined,
 
+  /**
+   * @override
```

Comment:
No, you’re right. @protected is what I wanted here.

I’ll add @override to the classes which inherit this.